File: server/EzySmashers-app-api/src/main/java/org/youngmonkeys/ezysmashers/app/model/PlayerHitModel.java
package org.youngmonkeys.ezysmashers.app.model;

import lombok.Builder;
import lombok.Getter;

@Getter
@Builder
public class PlayerHitModel {
    private float[] attackPosition;
    private int myClientTick;
    private int otherClientTick;
    private String victimName;

    public PlayerHitModel(
        float[] attackPosition,
        int myClientTick,
        int otherClientTick,
        String victimName
    ) {
        this.attackPosition = attackPosition;
        this.myClientTick = myClientTick;
        this.otherClientTick = otherClientTick;
        this.victimName = victimName;
    }
}

File: client-unity/Assets/2 - Scripts/models/PlayerHitModel.cs

using UnityEngine;

public class PlayerHitModel
{
	public string VictimName { get; set; }
	public Vector3 AttackPosition { get; set; }
	public int AttackerTick { get; set; }
	public int VictimTick { get; set; }

	public PlayerHitModel(string victimName, Vector3 attackPosition, int attackerTick, int victimTick)
	{
		VictimName = victimName;
		AttackPosition = attackPosition;
		AttackerTick = attackerTick;
		VictimTick = victimTick;
	}
}